### ISSUE TITLE
Accessibility Issue 1 (Input is missing a label)

### ISSUE THREAD

**Issue by yenyin262** [jagdeepsingh1990/Aloha_Project#2]
1.   Placeholder attribute may not be read by assistive technologies. In order to make it accessible you can include a <label> </label tag

ie. <label for="emailTextField">
  

- Enter email address
</label>
